.gitlab-ci.viz.yml

```diff
@@ -1,16 +1,13 @@
 stages:
   - test
 
-
```

Comment:
Why the removal of the blank lines? I added them for split clarity between jobs and env config.